test/integration/049_dbt_debug_test/test_debug.py

```diff
@@ -103,3 +104,27 @@ def test_postgres_badproject(self):
                 self.assertIn('ERROR invalid', line)
             elif line.strip().startswith('profiles.yml file'):
                 self.assertNotIn('ERROR invalid', line)
+
+    @use_profile('postgres')
+    def test_postgres_not_found_project_dir(self):
+        self.use_default_project()
```

Comment:
You shouldn't need to call `use_default_project` unless you do things with the project/profile as part of the test - `setUp` will call it (some old tests still have it, so I understand where this came from!)

Reply:
Yes, you're right @beckjake . I did not realize that it is called in `setUp` method. I will take it into account for the next time, thanks!